script.js

```diff
@@ -39,21 +41,40 @@ $(document).ready(function () {
         event.preventDefault();
 
         let data = getFormData($('form'));
-        
+
         createPdfDoc(data);
     });
 
     $(window).on('resize', resizeSignaturePad);
 
+    function copySameAddress(data) {
+        data.form_city = data.form_id_city;
+        data.form_county = data.form_id_county;
+        data.form_street = data.form_id_street;
+        data.form_street_no = data.form_id_street_no;
+        data.form_building = data.form_id_building;
+        data.form_building_entrance = data.form_id_building_entrance;
+        data.form_building_floor = data.form_id_building_floor;
+        data.form_appartment_no = data.form_id_appartment_no;
```

Comment:
Crezi ca am putea face, in cazul in care ai bifat `Locuiesc la aceeasi adresa, sa inlocuim cu valoarea` sa inlocuim cu  `"-"` sau cu `""` in toate campurile?
In subsolul paginii scrie ca daca locuiesti la aceeasi adresa, nu este nevoie sa completezi acea sectiune.
Mersi mult pentru ajutor!

Reply:
Done!